### DISCUSSION TITLE
Bot Hangs On First Link

### DISCUSSION THREAD

**Discussion by TeraHash** [LeonShams/BestBuyBulletBot#11] [General]
Hello Leon, 

The bot was functioning fine earlier, but now does nothing after a normal start or verifying account using the --verify-account command.

### Expected behavior
The bot should go through each link and save the cookies and price for each product.

### Actual behavior
The bot hangs, and eventually throws a WebDriverException, which says that it "Failed to decode response from marionette".

### To Reproduce
1. Run the bot for a few minutes normally, then exit by closing the cmd window.
2. Start the bot again using the start or --verify-account commands.

### Screenshots
https://imgur.com/a/KahAV6u

### System information
OS Platform: Windows 10 
BestBuyBulletBot version: 1.0.0b2
Python version: 3.9.7

**Comment by TeraHash** [comment]
Full stacktrace: https://pastebin.com/1qV2U8G0

**Comment by TeraHash** [comment]
I fixed this myself. As I had first thought, the CSS for the page had changed, so I had to change line 256 in browser.py from:

 ".pricing-price > div > div > div > .priceView-hero-price.priceView-customer-price",

to

".pricing-price > div > div > div > div > section > div > div > .priceView-hero-price.priceView-customer-price",

Hope this helps anyone who had the same issue as me. I will try to find a way to consistently locate the product price, as it is known already that websites will change their CSS to counteract bots, like Best Buy did to one of its buttons earlier today.

**Reply by LeonShams** [reply]
Thanks a lot for this! I have updated the bot to support the new CSS structure for item pricing.

**Comment by LeonShams** [comment]
To anyone else who sees this:

In the future please open bug reports in the issues tab. The discussions section is more for questions, help with usage or installation, or advice.
